AdyenComponents/ACH Direct Debit/ACHDirectDebitComponent.swift

```diff
@@ -349,7 +349,9 @@ public protocol AnyACHDirectDebitConfiguration {
 extension ACHDirectDebitComponent {
     
     /// Configuration for the ACH Direct Debit Component
-    public struct Configuration: AnyACHDirectDebitConfiguration, AnyPersonalInformationConfiguration {
+    public struct Configuration: AnyACHDirectDebitConfiguration, AnyPersonalInformationConfiguration, CheckoutComponentConfiguration {
```

Comment:
You chose a doubly annoying one to start 😄 
- We need to move this config to its own public area as the component itself is not public anymore. 
- We also need to remove the conformance of `AnyACHDirectDebitConfiguration`. This interface was added to allow both dropin and standalone components to have their own ACH configs. 
It was the same case with card config, we are getting rid of these interfaces, there will be one ACH config defined and will be used by both the component and dropin.